source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,588 @@
+---
+title: ovirt-provider-ovn security groups
+category: feature
+authors: mdbarroso
+feature_name: security-group-support
+feature_modules: engine,ovirt-provider-ovn
+feature_status: In Development
+---
+
+
+# Ovirt-provider-ovn security groups
+
+
+## Summary
+
+This feature aims to implement the OpenStack Networking API [security group](https://developer.openstack.org/api-ref/network/v2/index.html#security-groups-security-groups)
+in the ovirt-provider-ovn.
+
+By doing so, we allow fine-grained access control to - and from - the oVirt
+VMs attached to external networks.
+
+The Networking API v2 defines security groups as a white list of rules.
+That means, that by default, neither incoming nor outgoing traffic is
```

Comment:
I didn't understand how this relates to the previous line..
I think you can state it as a stand alone fact

Reply:
Since the rules are expressed as a white-list, you have to specify which traffic is allowed. In short, it literally is a 'list of allowed stuff'. As opposed to a black-list, where you list the things you block.

That means that if you don't white-list anything, your 'list of allowed stuff' is empty, thus making everything blocked by default.

Makes sense to you ?

Reply:
Now it does - but it's not so trivial to understand that having an entity such a security groups that is in fact a white list means that by default all traffic is not allowed.

Reply:
I've reworked it a bit; I am forced to agree I missed a premise there in the explanation.

Let me know if it's OK now.